components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/authz/OAuth2AuthzEndpoint.java

```diff
@@ -905,6 +911,8 @@ private String handleOAuthAuthorizationRequest(OAuthMessage oAuthMessage)
         OAuthAuthzRequest oauthRequest = new CarbonOAuthAuthzRequest(oAuthMessage.getRequest());
 
         OAuth2Parameters params = new OAuth2Parameters();
+        String sessionDataKey = UUIDGenerator.generateUUID();
```

Comment:
What is the correct UUID generator? UUIDGenerator comes from registry or java.util.UUID. Use one across all uuid generation instances?